### ISSUE TITLE
cant find correct guide for segment custom training 

### ISSUE THREAD

**Issue by jimmyflycv** [ultralytics/ultralytics#3718]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

Not so sure why does the training  force to download an unwanted pretrained model, i just follow this guide: https://docs.ultralytics.com/tasks/segment/#models

training log:
TensorBoard: Start with 'tensorboard --logdir runs/segment/train', view at http://localhost:6006/
AMP: running Automatic Mixed Precision (AMP) checks with YOLOv8n...
Downloading https://github.com/ultralytics/assets/releases/download/v0.0.0/yolov8n.pt to yolov8n.pt...

my training script is:
yolo task=segment train data=pvt.yaml model=yolov8m-seg.yaml pretrained=./yolov8m-seg.pt epochs=100 imgsz=640  batch=128

### Additional
./yolov8m-seg.pt is my local downloaded pretrained model

yolo/engine/trainer: task=segment, mode=train, model=yolov8m-seg.yaml, data=pvt.yaml, epochs=100, patience=20, batch=128, imgsz=640, save=True, save_period=-1, cache=False, device=0,1,2,3, workers=8, project=None, name=None, exist_ok=False, pretrained=./yolov8m-seg.pt, optimizer=auto, verbose=True, seed=0, deterministic=True, single_cls=False, rect=False, cos_lr=False, close_mosaic=10, resume=False, amp=True, fraction=1.0, profile=False, overlap_mask=True, mask_ratio=4, dropout=0.0, val=True, split=val, save_json=False, save_hybrid=False, conf=None, iou=0.7, max_det=300, half=False, dnn=False, plots=True, source=None, show=False, save_txt=False, save_conf=False, save_crop=False, show_labels=True, show_conf=True, vid_stride=1, line_width=None, visualize=False, augment=False, agnostic_nms=False, classes=None, retina_masks=False, boxes=True, format=torchscript, keras=False, optimize=False, int8=False, dynamic=False, simplify=False, opset=None, workspace=4, nms=False, lr0=0.01, lrf=0.01, momentum=0.937, weight_decay=0.0005, warmup_epochs=3.0, warmup_momentum=0.8, warmup_bias_lr=0.1, box=7.5, cls=0.5, dfl=1.5, pose=12.0, kobj=1.0, label_smoothing=0.0, nbs=64, hsv_h=0.015, hsv_s=0.7, hsv_v=0.4, degrees=0.0, translate=0.1, scale=0.5, shear=0.0, perspective=0.0, flipud=0.0, fliplr=0.5, mosaic=1.0, mixup=0.0, copy_paste=0.0, cfg=None, v5loader=False, tracker=botsort.yaml, save_dir=runs/segment/train
_No response_

**Comment by jimmyflycv** [comment]
AMP: running Automatic Mixed Precision (AMP) checks with YOLOv8n...

Is this automatically downloaded model just for amp test?

**Comment by glenn-jocher** [comment]
@jimmyflycv hi there! The automatically downloaded model (`yolov8n.pt`) is not just for AMP (Automatic Mixed Precision) tests. This model is actually a pretrained checkpoint that serves as a starting point for training the YOLOv8n model. During training, this checkpoint is loaded and the model is fine-tuned on your custom dataset.

The purpose of including this pretrained model is to provide a good initialization for the YOLOv8n model, which can improve training performance and convergence. However, if you already have a pretrained model (`yolov8m-seg.pt`) that you want to use instead, you can specify its path in the `pretrained` argument of the training script. This will skip the download of the default pretrained model and use your local pretrained model instead.

I hope this clarifies the purpose of the automatically downloaded model. Let me know if you have any further questions or concerns!
